packages/components/src/Disclosure/Disclosure.css

```diff
@@ -1,14 +1,15 @@
 .arrowIconWrapper {
-  transform-origin: 50% 50% 0;
+  transform-origin: 50% 47.5% 0;
```

Comment:
Makes the arrow appear to flip on its' _visual_ y-axis center, not its' "real" y-axis center (arrows are weird optically)